### PR TITLE
Add mixpanel for logging stoprestarts

### PR DESCRIPTION
**Pull request by wrefgtzweve**

### CODE REVIEW

File: lua/ulx/modules/sh/cfc_stoprestart.lua [hunk 1]
```diff
@@ -17,6 +26,7 @@ function cmd.tryStop( caller )
         return
     end
 
+    mixpanelTrackPlyEvent( "Restart stopped", ply )
```
Comment (brandonsturgeon) [inline lua/ulx/modules/sh/cfc_stoprestart.lua:29]:
```suggestion
    mixpanelTrackPlyEvent( "Player stopped automatic restart", ply )
```

**Review by brandonsturgeon — CHANGES_REQUESTED** [review]
Might be nice to include some data about how many people are on the server and how many in a row this was or something.

**Comment by wrefgtzweve** [comment]
Good idea, will add

**Comment by brandonsturgeon** [comment]
Syntax has changed a bit for Mixpanel - once that's updated we can merge

**Comment by wrefgtzweve** [comment]
Ready for review

**Review by brandonsturgeon — APPROVED** [review]

**Pull request merged**